src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/SleepInterval.java

```diff
@@ -0,0 +1,33 @@
+package pl.tomaszdziurko.jvm_bloggers.mailing;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.ToString;
+
+import java.util.concurrent.TimeUnit;
+
+@AllArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class SleepInterval {
```

Comment:
@Data was omitted intentionally to introduce methods `asMiliseconds` and `asSeconds` which read better `sleepInterval.asMilliseconds()` is easier to read than `sleepInterval.getMilliseconds()`